cmake/windows_dll_exports.def.in

```diff
@@ -60,3 +60,7 @@ PaUtil_SetDebugPrintFunction        @55
 @DEF_EXCLUDE_WASAPI_SYMBOLS@PaWasapiWinrt_SetDefaultDeviceId    @67
 @DEF_EXCLUDE_WASAPI_SYMBOLS@PaWasapiWinrt_PopulateDeviceList    @69
 @DEF_EXCLUDE_WASAPI_SYMBOLS@PaWasapi_GetIMMDevice               @70
+@DEF_EXCLUDE_WMME_SYMBOLS@PaWinMME_GetStreamInputHandleCount  @71
+@DEF_EXCLUDE_WMME_SYMBOLS@PaWinMME_GetStreamInputHandle       @72
+@DEF_EXCLUDE_WMME_SYMBOLS@PaWinMME_GetStreamOutputHandleCount @73
+@DEF_EXCLUDE_WMME_SYMBOLS@PaWinMME_GetStreamOutputHandle      @74
```

Comment:
PRs should address a single issue, therefore these four symbols should be added in a separate PR that also updates build/msvc/portaudio.def

Reply:
You're asking me to undo this only to have to redo it later after #503 is merged? That doesn't make much sense.

Reply:
No, I'm asking you to move it to another PR. We can merge the other PR ASAP. The point is: don't put unrelated changes in a single commit or PR.

Reply:
This pedantry about process is already frustrating enough. Please don't merge #503 before this and create even more work for me after I tried to proactively fix the merge conflict. I removed this so it will have to be re-added to re-fix the bug later.

Reply:
Following standard industry practice for keeping unrelated changes in separate PRs is not "pedantry". 
Thank you for removing those .def changes. We can add them back in easily in another PR.
We will not merge #503 before this one.